apps/src/netsim/NetSimBitLogPanel.html.ejs

```diff
@@ -0,0 +1,52 @@
+<%
+  var netsimConstants = require('./netsimConstants');
+  var dataConverters = require('./dataConverters');
+
+  var getEncodingLabel = require('./netsimUtils').getEncodingLabel;
+
+  var EncodingType = netsimConstants.EncodingType;
+  var PacketUIColumnType = netsimConstants.PacketUIColumnType;
+
+  var formatAB = dataConverters.formatAB;
+  var formatBinary = dataConverters.formatBinary;
+  var formatHex = dataConverters.formatHex;
+  var alignDecimal = dataConverters.alignDecimal;
+  var binaryToAB = dataConverters.binaryToAB;
+  var binaryToHex = dataConverters.binaryToHex;
+  var binaryToDecimal = dataConverters.binaryToDecimal;
+  var binaryToAscii = dataConverters.binaryToAscii;
```

Comment:
Nit: Rather than predeclaring everything here and then using a single time later, it might make things more readable to just call dataConverters.foo later. Also, it looks like 5 of these 8 converters are actually used?